### ISSUE TITLE
Auto update not working

### ISSUE THREAD

**Issue by tommymsw1** [RuudBurger/CouchPotatoServer#4242]
### Steps to reproduce:
1. .. Press "Just update"
2. ..

Auto update used to work fine, then just stopped (I don't know how many versions ago)
### Information:
Movie(s) I have this with: ...
Quality of the movie being searched: ...
Providers I use: ...
Version of CouchPotato: desktop: 2.6.0
Running on: ...

### Logs:
```
11-23 12:23:31 ERROR [          couchpotato.api] Failed doing api request "updater.update": Traceback (most recent call last):
  File "C:\Users\NETTOP\AppData\Roaming\CouchPotato\application\appdata\CouchPotato-2.6.0.win32\couchpotato\api.py", line 35, in run_handler
  File "C:\Users\NETTOP\AppData\Roaming\CouchPotato\application\appdata\CouchPotato-2.6.0.win32\couchpotato\core\_base\updater\main.py", line 128, in doUpdateView
  File "C:\Users\NETTOP\AppData\Roaming\CouchPotato\application\appdata\CouchPotato-2.6.0.win32\couchpotato\core\_base\updater\main.py", line 100, in check
TypeError: a float is required 
```

**Comment by jc82** [comment]
Hello!
I'm having this issue. Bottom banner notifies that an update is available - not amount of clicking seems to matter. I was hoping that updating would solve an issue where CP no longer seems to be using my usenet provider to search/find movies...

Info:
 * Ubuntu 14.04.1 LTS (GNU/Linux 3.13.0-43-generic x86_64)
 * Server Edition - No GUI installed
 * CP Version  1fb031ff (10/12/2014 10:41:56 AM)
 * Updater  git, master
 *  Automatic Updates are turned on

**Comment by bthusby** [comment]
Yeah, I have the same problem on CP too on my Synology!

**Comment by Datacore2** [comment]
Confirming this issue. I tried clicking the orange update link and got the same error "TypeError: a float is required". I turned on automatic updates. Restarted couchpotato. Clicked the update link. It updated.

**Comment by jc82** [comment]
I gave up and reinstalled CP on my server. Now I can't get seem to get CP to start on boot, but I can live with that issue for now.

**Comment by RuudBurger** [comment]
This is fixed in version 2.6.3
